scripts/validate.sh

```diff
@@ -32,6 +32,13 @@ function _validate_component() {
     fi
 }
 
+function validate_semver() {
```

Comment:
This isn’t used. (But `validate_semver` is a better name than `is_semver` for a function which exits if the validation fails.)

Reply:
Right good cathc, I was thinking to move it but decided to leave in utils and forgot this copy :)